### PR TITLE
Adding cards for single production of B>bH for Run3

### PR DESCRIPTION
**Pull request by DaLu0**
Dears,
My name is Dalia Ramírez from Universidad Iberoamericana (México). We, together with my supervisor Cristina Oropeza, are working with the B2G group.
We would like to produce the samples for our process for Run3 using the model VLQ_v4_4FNS_UFO, using as starting point the following process from Run2:
https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/bpbj_LO_M2200_100k_narrow_Single

Could you please help us by telling if there is something to be changed or modified.

Kind regards,
Dalia

### CODE REVIEW

**Comment by acarvalh** [comment]
Hi @DaLu0 

- The updates that need to be made are summarized in the commits from this [PR](https://github.com/cms-sw/genproductions/pull/3447)

To the reviewers, I suggested an organization to single VLQ cards in this [PR](https://github.com/cms-sw/genproductions/pull/3470) to help with incoming cards and also with the bookkeeping of the channels we need to cover

**Comment by DaLu0** [comment]
Dear @acarvalh 
Thanks a lot for your reply.
The cards were added to the /cards/production/13p6TeV/ directory, is this correct?
The beam energy was updated to 6800 GeV.
And, as we are not using madspin card, I think that was the only change to be done.

Please let me know if there is any other comment.
Cheers,
Dalia

**Comment by acarvalh** [comment]
Hi @DaLu0 

To me, it seems correct, but the review of the settings is the area of the GEN folks ;-)
 
I have some more minor comments on the naming convention of the folders, also to allow an easier finding of those afterward, 

- could you rename the folders to match the conventions used on that first [PR](https://github.com/cms-sw/genproductions/pull/3470/commits) we had for single VLQ?  That could be: `bpbj_R3_LO_M600_100k` --> `BprimeBToBH_LH_narrowLO` (please check if my suggestion makes sense and adapt accordingly if not, specially the 'LH')

- What does R3 stand for?

- You do not need to add the number of events in the folder name

The real best would be that the reviewers accept my MR to rearrange folders and you add the cards to the subfolder dedicated to the single VLQ

**Comment by acarvalh** [comment]
Hi @DaLu0 
[This](https://github.com/cms-sw/genproductions/pull/3470) is merged (thank you @menglu21 !). So please on renaming also move the cards to `genproductions/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/single_VLQ`

Provided that the settings are agreed by GEN folks we are good to go!

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bpbj_R3_LO_M600_100k/bpbj_R3_LO_M600_100k_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import model VLQ_v4_4FNS_UFO-3rdL_noCKM
+set complex_mass_scheme
+define q = u c d s u~ c~ d~ s~
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/bpbj_R3_LO_M600_100k/bpbj_R3_LO_M600_100k_proc_card.dat:9]: those lines can be removed

**Comment by DaLu0** [comment]
Dear all, sorry for my delay.
I have just added the cards to the corresponding directory (singleVLQ) and created the directory "BprimeBToBH_LH_narrowLO" the cards were corrected, can you please tell me if this is correct?

**Comment by menglu21** [comment]
Hi @DaLu0 it seems your cards are not in genproductions/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/single_VLQ.

**Comment by bbilin** [comment]
I am closing the PR, since its long pending. If it is needed please reopen.

**Pull request closed without merging**